CMakeLists.txt

```
@@ -435,6 +435,10 @@ if(WITH_CORE)
   # required
   find_package(Proj REQUIRED)
   message(STATUS "Found Proj: ${PROJ_VERSION} ${PROJ_DIR}")
+  if(PROJ_VERSION_MAJOR LESS 8 OR (PROJ_VERSION_MAJOR EQUAL 8 AND (PROJ_VERSION_MINOR LESS 1)))
+    message(FATAL_ERROR "Cannot build QGIS using Proj older than 8.1")
+  endif(PROJ_VERSION_MAJOR LESS 8 OR (PROJ_VERSION_MAJOR EQUAL 8 AND (PROJ_VERSION_MINOR LESS 1)))
+
```

wouldn't the following work ?
```suggestion
  if(PROJ_VERSION VERSION_LESS "8.1")
    message(FATAL_ERROR "Cannot build QGIS using Proj older than 8.1")
  endif()

```

---

Indeed, that's much better!